shardingsphere-features/shardingsphere-authority/shardingsphere-authority-api/src/main/java/org/apache/shardingsphere/authority/model/ShardingSpherePrivileges.java

```diff
@@ -23,28 +23,28 @@
  * ShardingSphere Privileges.
  */
 public interface ShardingSpherePrivileges {
-    
+
     /**
      * Set super privilege.
      */
     void setSuperPrivilege();
-    
```

Comment:
Hi @totalo,

Thanks for your efforts. It looks much better.
Just one issue here.  There is no necessary changes in this file, I guess,  could you  revert it?

Reply:
ok~